pkg/autopilot/controller/signal/k0s/restart_windows.go

```diff
@@ -32,6 +32,6 @@ func restartEventFilter(hostname string, handler apsigpred.ErrorHandler) crpred.
 //
 // This controller is only interested in changes to signal nodes where its signaling
 // status is marked as `Restart`
-func registerRestart(logger *logrus.Entry, mgr crman.Manager, eventFilter crpred.Predicate, delegate apdel.ControllerDelegate) error {
+func registerRestart(logger *logrus.Entry, mgr crman.Manager, eventFilter crpred.Predicate, delegate apdel.ControllerDelegate, _ string) error {
```

Comment:
In this case, I think it makes the code easier to follow (and more symmetric with the non-windows code) when the parameter names are spelled out, even if they're unused. (A string might be used for pretty much anything and doesn't give much context as a type on its own.)